setup.py

```diff
@@ -99,7 +99,7 @@ def run(self):
                         'base58',
                         # pinned because issue with fpm from v4.0.0
                         'importlib_metadata==3.10.1',
-                        # 'ioflo==2.0.2',
+                        'ioflo==2.0.3',
```

Comment:
This will work, in that the expected version of `ioflo` will be installed, however `ioflo` is a dev/test dependency and not a runtime dependency.  Therefore it should not be defined here.  It should be defined in the `tests_require` section starting on line 30.